internal-client/src/main/java/com/tencent/rss/client/response/RssGetShuffleDataResponse.java

```diff
@@ -18,21 +18,20 @@
 
 package com.tencent.rss.client.response;
 
-import com.tencent.rss.common.ShuffleDataResult;
-
 public class RssGetShuffleDataResponse extends ClientResponse {
 
-  private ShuffleDataResult shuffleDataResult;
+  private byte[] shuffleData;
 
   public RssGetShuffleDataResponse(ResponseStatusCode statusCode) {
     super(statusCode);
   }
 
-  public ShuffleDataResult getShuffleDataResult() {
-    return shuffleDataResult;
+  public byte[] getShuffleData() {
+    return shuffleData;
   }
 
-  public void setShuffleDataResult(ShuffleDataResult shuffleDataResult) {
-    this.shuffleDataResult = shuffleDataResult;
+  public void setShuffleData(byte[] data) {
```

Comment:
remove setShuffleData and add a parameter named shuffleDataResult for constructor？

Reply:
OK